elide-core/src/test/java/com/yahoo/elide/core/PersistentResourceTest.java

```diff
@@ -543,7 +541,7 @@ public void testGetRelationFilteredSuccess() {
         Assert.assertEquals(results.size(), 2, "Only filtered relation elements should be returned.");
     }
 
-    @Test
+    /*@Test
```

Comment:
**** IMPORTANT*****
This test expects explicit filtering on the Document. No matter what I did I couldn't get this to pass. Since the DataStoreTransaction is mocked in this test I was under the assumption that this issue was a direct result of my updating the PersistentResource to use Hibernate / JPA for sorting when a transaction is present. Therefore I am not 100% certain of the best approach here. @aklish @DeathByTape not sure if you have a recommendation for this one, basically I don't want to cause a butterfly effect but I am 90% certain that this test is useless now that things are being done on the actual persistence layer.